netlify.toml

```diff
@@ -0,0 +1,8 @@
+[build]
+  command = 'yarn build-docz'
```

Comment:
The Netlify build fails because the following command is not matching the build task name from `package.json`
```suggestion
  command = 'yarn docz:build'
```

Reply:
Ha, thanks! Missed that it changed.